github_api.py

```diff
@@ -22,7 +22,9 @@ def get_open_pull_request():
 
     pull_requests = {}
     # Download open pull requests
-    for owner, repo_name in github_projects:
+    for metadata in github_projects:
```

Comment:
Namedtuple is still a tuple so these for loops don't have to be changed at all and you can use the previous unpacking.

Reply:
ok, this part reverted